ecommerce/products/base.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+
+class Rarity:
+    COMMON = 'common'
+    UNCOMMON = 'uncommon'
+    RARE = 'rare'
+    ULTRA_RARE = 'ultra rare'
+    AMAZING_RARE = 'amazing rare'
+    SECRET_RARE = 'secret rare'
+
+class Languages:
+    ENGLISH = 'English'
+    JAPANESE = 'Japanese'
+    FRENCH = 'French'
+    SPANISH = 'Spanish'
+    GERMAN = 'German'
+    CHINESE ='Chinese'
+    KOREAN = 'Korean'
+
+class Condition:
+    SEALED_PRODUCT = 'sealed product'
+    PACK_FRESH = 'pack fresh'
+    LIGHT_PLAY = 'light play'
+    MODERATE_PLAY = 'moderate play'
+    HEAVY_PLAY = 'heavy play'
+    DAMAGED = 'damaged'
+
+class ProductType:
+    BOOSTER_BOX = 'booster box'
+    ELITE_TRAINER_BOX_PLUS = 'elite trainer box plus'
+    ELITE_TRAINER_BOX = 'elite trainer box'
+    PREMIUM_COLLECTION_BOX 'premium collection box'
+    POKEMON_THEMED_BOX = 'pokemon themed box'
+    PIN_BOX = 'pin box'
+    THEME_DECK = 'theme deck'
+    BUILD_AND_BATTLE_DECK = 'build and battle deck'
+    PREMIUM_BLISTER = 'premium blister'
+    THREE_PACK_BLISTER = 'three pack blister'
+    SINGLE_PACK_BLISTER = 'single pack blister'
+    BOOSTER_PACK = 'booster pack'
+    SINGLE_CARD = 'single card'
```

Comment:
Do you think this list is likely to change? If it is something that would change regularly then we might want to consider making `ProductType` a model of its own. But if you're fairly sure this won't need to change much then let's just leave it as a choice field for now.

Reply:
Nah i reckon this is a pretty concrete list of all product types that they release. I'm pretty sure i covered every possible type of product they might release with a set.